spec/lib/data_update_scripts/update_default_email_addresses_spec.rb

```diff
@@ -4,20 +4,8 @@
 )
 
 describe DataUpdateScripts::UpdateDefaultEmailAddresses do
-  let(:contact_email) { "contact@dev.to" }
-
-  context "with no contact email set in the email_addresses hash" do
-    it "adds the contact email" do
-      described_class.new.run
-      expect(Settings::General.email_addresses).to include(contact: ApplicationConfig["DEFAULT_EMAIL"])
-    end
-  end
-
-  context "with a contact email set in the email_addresses hash" do
-    it "preserves the current contact email" do
-      Settings::General.email_addresses[:contact] = contact_email
-      described_class.new.run
-      expect(Settings::General.email_addresses).to include(contact: contact_email)
-    end
+  it "adds the default email" do
+    described_class.new.run
```

Comment:
When I was testing the specs in random order, I saw this fail (on main), because the update script has an early exit if the email was already set. It might make sense to delete the `:default` key from the email addresses if present, before running the script, to ensure we're in the state we expect for this job to be effective.

I didn't bisect how the test order led to that situation, and we don't run randomly by default, and it _may_ have been just the two prior contexts executing bottom up instead of top down (we set the contact email, assert nothing changes, assume there was no contact email set, assert expected change - fails). If that's the case, only having the one path here reduces the likelihood of failure.

Does this fail if just the two specs `spec/lib/data_update_scripts/remove_unused_site_config_emails_spec.rb` (populates email address settings) and `spec/lib/data_update_scripts/update_default_email_addresses_spec.rb` (expects email address settings to be empty) are run in order? 

I tested - *no they don't fail* - I think database isolation between specs is working fine and it must have been out-of-order execution of the original test (which is no longer an issue because there is a single example here now).